modules/gridBidAdapter.js

```diff
@@ -44,19 +43,237 @@ export const spec = {
    * @return {ServerRequest[]} Info describing the request to the server.
    */
   buildRequests: function(validBidRequests, bidderRequest) {
-    const oldFormatBids = [];
-    const newFormatBids = [];
+    if (!validBidRequests.length) {
+      return null;
+    }
+    let pageKeywords = null;
+    let jwpseg = null;
+    let content = null;
+    let schain = null;
+    let userId = null;
+    let user = null;
+    let userExt = null;
+    let {bidderRequestId, auctionId, gdprConsent, uspConsent, timeout, refererInfo} = bidderRequest || {};
+
+    const referer = refererInfo ? encodeURIComponent(refererInfo.referer) : '';
+    const imp = [];
+    const bidsMap = {};
+
     validBidRequests.forEach((bid) => {
-      bid.params.useNewFormat ? newFormatBids.push(bid) : oldFormatBids.push(bid);
+      if (!bidderRequestId) {
+        bidderRequestId = bid.bidderRequestId;
+      }
+      if (!auctionId) {
+        auctionId = bid.auctionId;
+      }
+      if (!schain) {
+        schain = bid.schain;
+      }
+      if (!userId) {
+        userId = bid.userId;
+      }
+      const {params: {uid, keywords, bidFloor}, mediaTypes, bidId, adUnitCode, realTimeData} = bid;
+      bidsMap[bidId] = bid;
+      if (!pageKeywords && !utils.isEmpty(keywords)) {
+        pageKeywords = utils.transformBidderParamKeywords(keywords);
+      }
+      if (realTimeData && realTimeData.jwTargeting) {
+        if (!jwpseg && realTimeData.jwTargeting.segments) {
+          jwpseg = realTimeData.jwTargeting.segments;
+        }
+        if (!content && realTimeData.jwTargeting.content) {
+          content = realTimeData.jwTargeting.content;
+        }
+      }
+      let impObj = {
+        id: bidId,
+        tagid: uid.toString(),
+        ext: {
+          divid: adUnitCode
+        }
+      };
+
+      if (bidFloor) {
+        impObj.bidfloor = bidFloor;
```

Comment:
Please update to support the [Price Floor module ](https://docs.prebid.org/dev-docs/modules/floors.html)as per other bidders and default to the bidFloor param value if no Floor value is passed in the module. You can refer to other bidders for example of how to implement.

Reply:
Fixed